slices/libasan6.yaml

```diff
@@ -0,0 +1,9 @@
+package: libasan6
+
+slices:
+  libs:
+    essential:
```

Comment:
the dependencies should be `libc6`, `libgcc-s1` and `gcc-11-base` instead.

Reply:
Added the `libgcc-s1` dependency. I am unable to find `gcc-11-base`, in the slices folder? Do I miss something? 😅

Reply:
It doesn't exist yet. 

**However**, since it is only a collection of `/usr/share/doc/` files, it isn't really needed for the slice you're proposing. So I recommend simply adding a comment saying precisely that - "Although `gcc-11-base` is a package dependency, it is not needed for this `libs` slice"

Reply:
Alrighty, done :)